homeassistant/components/hydrawise/sensor.py

```diff
@@ -99,4 +97,4 @@ def _update_attrs(self) -> None:
             if (next_run := self.zone.scheduled_runs.next_run) is not None:
                 self._attr_native_value = dt_util.as_utc(next_run.start_time)
             else:
-                self._attr_native_value = datetime.max.replace(tzinfo=dt_util.UTC)
+                self._attr_native_value = None
```

Comment:
I think that this would cause the sensor to be unavailable instead of being unknown

Reply:
No, unavailable is a separate property. `None` is a perfectly valid value and converts to "unknown". See the test case.

Reply:
Check this discussion as I think it's equivalent to this. https://github.com/home-assistant/core/pull/104522#discussion_r1477508484

Reply:
Oh, I misunderstood your original comment.

I don't think setting this to unavailable is correct. 

https://developers.home-assistant.io/docs/core/entity#generic-properties says "Indicate if Home Assistant is able to read the state and control the underlying device."

In this case, we can read the sensor value just fine--it just doesn't *have* a value. That's equivalent to `None`, which equates to "unknown".

Reply:
Ok! I will leave it up to another member to determine this :)

Reply:
If you're not comfortable reviewing this, can you unassign yourself so someone else can review?